### ISSUE TITLE
Sample database

### ISSUE THREAD

**Issue by bavarialogy** [Ashutoshcoder/Attendance-Manager-Website-master#1]
Hello there,
I just tried to set this up but I'm lacking a database to start with. Is there a template somewhere?

**Comment by rowan-sandhu** [comment]
Yes, I also need the same. Please provide a database file or tell how to setup database for this project. Thanks in advance
